freedomofpress/securedrop-client
# fix placeholder label->widget

# Description

Fixes issue that was a result of this morning's merge: placeholder widget should have been `QWidget` instead of `QLabel` since we add two label child `QLabel` widgets. If we add two child `QLabel` widgets to a `QLabel` then it is not resized properly.

# Test Plan

1. Log in offline mode
2. Select a source and see placeholder now appears (confirm it's missing on `master`)

## Regression test

1. Log in with credentials
2. Select a source and see placeholder appears correctly
3. Log out and see placeholder appears correctly

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:


If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:


If these changes modify the database schema, you should include a database migration. Please check as applicable:
